src/ExchangeContainer/ExchangeContainer.js

```diff
@@ -1,17 +1,65 @@
 import React, { useState, useEffect } from 'react'
 import { Route } from 'react-router-dom'
-import { Link, NavLink } from 'react-router-dom';
-import './ExchangeContainer.scss';
+import { Link, NavLink } from 'react-router-dom'
+import './ExchangeContainer.scss'
 import CurrencyCard from '../CurrencyCard/CurrencyCard.js'
 import Bookmarked from '../Bookmarked/Bookmarked.js'
+import { getExchangeRates } from '../apiCalls.js'
+import { sampleApiData } from '../sampleApiData.js'
+ 
+const ExchangeContainer = ({ currencyInputData }) => {
+  const [exchangeRate, setExchangeRate] = useState(0)
+  const [rateDay, setRateDay] = useState('')
+  const [newAmount, setNewAmount] = useState(0)
+  const [allCurrencyData, setAllCurrencyData] = useState([])
+
+
+  useEffect(() => {
+    if(exchangeRate === '') {
+      // getExchangeRates(currencyInputData.userCurrency, currencyInputData.newCurrency)
+      // .then(data => setExchangeRate(data.rates[currencyInputData.newCurrency]))
+      setExchangeRate(0.7336110195)
```

Comment:
I don't think I'll need this now that I have the condition to keep it from running repeatedly!